src/model_parser.cc

```diff
@@ -334,6 +334,11 @@ ModelParser::InitOpenAI(
     delay_input->second.datatype_ = "UINT64";
     delay_input->second.shape_.push_back(1);
     delay_input->second.is_optional_ = true;
+
+    auto session_id_input = inputs_->emplace("session_id", ModelTensor()).first;
+    session_id_input->second.name_ = "session_id";
+    session_id_input->second.datatype_ = "BYTES";
+    session_id_input->second.shape_.push_back(1);
```

Comment:
As shown [here](https://github.com/triton-inference-server/perf_analyzer/pull/293/files#r1965109842), session ID is now treated as a top-level "inference input", and is no longer nested inside the `payload` "inference input" (which is also declared a few lines above this).